From 3b0ae804550321a7d7ab342b0d98c7b5badcd6ba Mon Sep 17 00:00:00 2001
From: Haoran Wang <b50027@freescale.com>
Date: Thu, 19 Nov 2015 20:17:22 +0800
Subject: [PATCH 21/22] MA-7125-1 Remove specific vendor codes in
 wpa_supplicant

As the in mashmallow 6 many bugs for supplicant have
been fixed. So try to remove the specific vendor codes
and validate the quality.

Signed-off-by: Haoran Wang <b50027@freescale.com>
---
 wpa_supplicant/Android.mk | 1 -
 1 file changed, 1 deletion(-)

diff --git a/wpa_supplicant/Android.mk b/wpa_supplicant/Android.mk
index a0efe23..32e3c82 100644
--- a/wpa_supplicant/Android.mk
+++ b/wpa_supplicant/Android.mk
@@ -1608,7 +1608,6 @@ LOCAL_STATIC_LIBRARIES += libnl_2
 endif
 endif
 LOCAL_CFLAGS := $(L_CFLAGS)
-LOCAL_CFLAGS += -DREALTEK_WIFI_VENDOR
 LOCAL_SRC_FILES := $(OBJS)
 LOCAL_C_INCLUDES := $(INCLUDES)
 include $(BUILD_EXECUTABLE)
-- 
1.8.0

